src/VirtualClient/VirtualClient.Core.UnitTests/Proxy/ProxyBlobManagerTests.cs

```diff
@@ -187,36 +182,9 @@ public Task ProxyBlobManagerDownloadsTheExpectedBlobThroughTheProxyApi_Scenario2
                     Assert.AreEqual("anypackage.1.0.0.zip", blobDescriptor.BlobName);
                     Assert.AreEqual("application/octet-stream", blobDescriptor.ContentType);
                     Assert.AreEqual(Encoding.ASCII.WebName, blobDescriptor.ContentEncoding);
-                })
-                .ReturnsAsync(this.mockFixture.CreateHttpResponse(System.Net.HttpStatusCode.OK));
-
-                return blobManager.DownloadBlobAsync(descriptor, this.mockStream, CancellationToken.None);
-            }
-        }
-
-        [Test]
-        public void ProxyBlobManagerThrowsIfTheBlobDownloadFails()
```

Comment:
Is there a reason you are changing this behavior? Please check Bryan's latest PR on proxy blob manager changes

Reply:
added back in